api/src/main/resources/templates/pdfviewer/pdfviewer.vm

```diff
@@ -35,7 +35,12 @@
   ## parameter, since an attachment picker is used, or by using both the file and document parameters, for macros added
   ## in xwiki versions before 11.5.
   #if ($file.startsWith('http://') || $file.startsWith('https://'))
-    #set ($url = $file)
+    #set ($attachRef = $services.model.resolveAttachment($file, "resource/standardURL"))
+    #set ($docRef = $attachRef.getDocumentReference())
+    #set ($fileName = $attachRef.getName())
```

Comment:
Since you use this variable only once, you could use the method call directly below, if this is not breaking the 120characters limit

Reply:
If I'm using the method call instead of both variables, it breaks the limit (134ch). The only variable that can be replaced with the call without breaking the line limit would be fileName, and the if would look like this:     
`#if ("$attachRef.getName()" == "$file" || $services.security.authorization.hasAccess('view', $docRef))`
Would this be ok?

Reply:
Yes, looks better IMO

Reply:
Ok, I'm committing the modification now